src/commands/app/deploy.ts

```diff
@@ -61,7 +61,8 @@ export default class AppDeploy extends AuthenticatedCommand {
   public async run(): Promise<void> {
     try {
       const { flags } = await this.parse(AppDeploy);
-      const { directoryPath, appId, appVersionId, region, force } = flags;
+      const { directoryPath, appId, appVersionId, region: strRegion, force } = flags;
+      const region = getRegionFromString(strRegion);
```

Comment:
Can also simplify things by getting rid of redundant const `regionStr`
```suggestion
      const { directoryPath, appId, appVersionId, force } = flags;
      const region = getRegionFromString(flags?.region);
```